packages/trader/src/Modules/Trading/Components/Form/TradeParams/Duration/duration.jsx

```diff
@@ -194,7 +194,7 @@ const Duration = ({
                             simple_duration_unit={simple_duration_unit}
                         />
                     )}
-                    {contract_type !== 'vanilla' && (
+                    {contract_type !== VANILLALONG.CALL && contract_type !== VANILLALONG.PUT && (
```

Comment:
same as https://github.com/binary-com/deriv-app/pull/9858/files#r1310999046

Reply:
Yep, will work. In this case I'll import `isVanillaContract`